examples/sync-api/src/utils/generate-entities.ts

```diff
@@ -0,0 +1,26 @@
+import c from 'chance';
+const chance = new c();
+
+interface Entity {
+  _key: string;
+  _class: string;
+  _type: string;
+  displayName: string;
+  from: string;
+}
+
+export const generateEntity = (): Entity => {
+  const packageName = chance.word();
+
+  return {
+    _key: `npm_package:${packageName}`,
+    _class: 'CodeModule',
+    _type: 'npm_package',
+    displayName: packageName,
+    from: 'testing',
+  };
+};
+
+export const generateNumberOfEntities = (records: number): Entity[] => {
+  return [...Array(records)].map(generateEntity);
```

Comment:
I don't think you need the spread here, just map the array.

Reply:
I thought the same thing but you actually do need the spread. `Array(records)` returns an array with no items, but rather 5 empty slots, so mapping over it does nothing.